fedora_messaging/tests/unit/test_message.py

```diff
@@ -22,11 +22,44 @@
 import jsonschema
 import mock
 
-from fedora_messaging import message
+from fedora_messaging import message, exceptions
+
+
+class GetMessageTests(unittest.TestCase):
+    """Tests for the :func:`fedora_messaging.message.get_message` function."""
+
+    def test_missing_severity(self):
+        """Assert the default severity is INFO if it's not in the headers."""
+        msg = message.Message(severity=message.ERROR)
+        del msg._headers["fedora_messaging_severity"]
+
+        recv_msg = message.get_message("", msg._properties, b"{}")
+        self.assertEqual(recv_msg.severity, message.INFO)
+
+    def test_invalid_severity(self):
+        """Assert the invalid severity fails validation."""
+        msg = message.Message()
+        msg._headers["fedora_messaging_severity"] = 42
+
+        self.assertRaises(
+            exceptions.ValidationError, message.get_message, "", msg._properties, b"{}"
+        )
+
+    def test_missing_headers(self):
+        """Assert missing headers results in a default message."""
+        msg = message.Message()
+        msg._headers = None
+        expected_message = message.Message()
+        expected_message.id = msg.id
```

Comment:
You don't seem to be using this variable.

Reply:
Ah, good eye